styleguide/components/21-atoms/field-message/README.md

```diff
@@ -0,0 +1,12 @@
+# Field message
+
+## General
+
+A field message is used in the context of a form or an interactive field.
+It's used as a sort of tooltip to indicate if a form field has been
+successfully filled in or not and should always be used in combination
+with a field.
+
+## Web accessibility
+
+* A field message should always have a visually hidden title.
```

Comment:
See before.
What we should do is 
* change the aria-describedby attribute on the affected field from the field description to the field message.
* add aria-invalid=true to the affected field if indeed invalid.
* add aria-role=alert to the field message, so changes are automatically read in case of ajax or frontend validation.